source/update.rst

```diff
@@ -0,0 +1,57 @@
+.. _operator-updates:
+
+Update Percona Distribution for PostgreSQL Operator
```

Comment:
1. We should mention smth about major version upgrades (from pg 12 to pg 13) - if not supported, we need to state it clearly.
2. I understand that we have only 2 versions of the operator, but once 1.1.0 is released - can the user upgrade from 0.2.0 to 1.1.0 directly? We need to mention that it is one version up upgrade only.